tests/ops/test_contract.py

```diff
@@ -401,6 +402,28 @@ def test_ubersum(equation, batch_dims):
                          output, expected_part.detach().cpu(), actual_part.detach().cpu()))
 
 
+@pytest.mark.parametrize('equation,batch_dims', UBERSUM_EXAMPLES)
+def test_ubersum_jit(equation, batch_dims):
+    inputs, outputs, operands, sizes = make_example(equation)
+
+    try:
+        expected = ubersum(equation, *operands, batch_dims=batch_dims, modulo_total=True)
+    except NotImplementedError:
+        pytest.skip()
+
+    @pyro.ops.jit.trace
+    def jit_ubersum(*operands):
+        return ubersum(equation, *operands, batch_dims=batch_dims, modulo_total=True)
+
+    actual = jit_ubersum(*operands)
+
+    if not isinstance(actual, tuple):
+        pytest.xfail(reason="https://github.com/pytorch/pytorch/issues/14875")
```

Comment:
Do we want to xfail here or just patch `pyro.ops.jit.trace` until the PyTorch issue is resolved?